lib/open_project/backlogs/engine.rb

```diff
@@ -132,10 +132,57 @@ def self.settings
       :ProjectsController, :ProjectsHelper, :Query, :User, :VersionsController, :Version]
 
     extend_api_response(:v3, :work_packages, :work_package) do
-      property :story_points, if: -> (*) { backlogs_enabled? }
-      property :remaining_hours, if: -> (*) { backlogs_enabled? }
+      property :story_points,
+               render_nil: true,
+               if: -> (*) { backlogs_enabled? }
```

Comment:
@NobodysNightmare I am aware that this bug existed before but right now story_points/remaining_time is displayed for every work package within a project that has backlogs enabled. But not every work package should have the attributes. Only work packages of types which backlogs considers to be stories/tasks should have those attributes. E.g. backlogs is configured to have "User story" and "Bug" as "Stories" and "To Do" as "Task" - then work packages of type "Phase" should not have those fields. 

This would then also apply to the front end where I currently cannot assess whether the fields are added project based or project + wp type based. Maybe you have a better insight into that @0xF013.